.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+sudo: required
+before_script:
+- npm install -g polymer-cli
+- polymer install --variants
+node_js: stable
+addons:
+  firefox: latest
+  apt:
+    sources:
+    - google-chrome
+    packages:
+    - google-chrome-stable
+  sauce_connect: true
+script:
+- polymer test -l chrome -l firefox
```

Comment:
don't need the `-l chrome -l firefox` as they are the only ones you have installed above.